README.md

```diff
@@ -65,23 +65,28 @@ Once you have a Census API key, you can configure a `CensusCredentials` block in
 
 ### Write and run a flow
 
-#### **Trigger Census sync run and wait for completion**:
+
+#### Use a `CensusSync` job block to run a sync and wait for completion
 
 ```python
 from prefect import flow
+from prefect_census import (
+    CensusCredentials, CensusSync, run_census_sync
```

Comment:
```suggestion
   CensusSync, run_census_sync
```

Reply:
why? the `CensusCredentials` are being used in the example below

Reply:
I see now; apologize for the noise.